pkg/broker/filter/stats_reporter.go

```diff
@@ -185,16 +196,25 @@ func (r *reporter) ReportEventProcessingTime(args *ReportArgs, err error, d time
 	return nil
 }
 
-func (r *reporter) generateTag(args *ReportArgs, t tag.Mutator) (context.Context, error) {
+func (r *reporter) generateTag(args *ReportArgs, tags ...tag.Mutator) (context.Context, error) {
 	// Note that filterType and filterSource can be empty strings, so they need a special treatment.
-	return tag.New(
+	ctx, err := tag.New(
 		context.Background(),
 		tag.Insert(r.namespaceTagKey, args.ns),
 		tag.Insert(r.triggerTagKey, args.trigger),
 		tag.Insert(r.brokerTagKey, args.broker),
 		tag.Insert(r.triggerFilterTypeKey, valueOrAny(args.filterType)),
```

Comment:
We need a followup issue to fix this, the triggerFilter for any value should be turned into a string and set.

Reply:
issue: https://github.com/knative/eventing/issues/1795

Reply:
hmm, that is what the valueOrAny does. If the filter is empty (meaning empty), we are setting the value "any" to the tag... so that we can filter on it...
Was that the concern or I misunderstood?

Reply:
oh! my bad `tag.Insert(r.triggerFilterTypeKey, valueOrAny(args.filterType)),`

Reply:
thanks!